### PR TITLE
Update K8s readme 

### PR DESCRIPTION
**Pull request by mszhanyi**
Add Microsoft Azure AKS support

### CODE REVIEW

**Comment by nbcsm** [comment]
@maxluk

File: kubernetes/README.md [hunk 1]
```diff
@@ -3,35 +3,37 @@
...
+are not using AWS or Azure, please refer to your cloud/infrastructure provider's manual
+to setup a kubernetes cluster.
 
-> **NOTE**: EKS is not required to run this controller, 
+> **NOTE**: EKS/AKS is not required to run this controller,
 >  you can use other Kubernetes clusters.
 
-Use `eksctl` to create an Amazon EKS cluster. This process takes ~15 minutes. 
+#### Create your cluster
+##### Ceate Amazon EKS
```
Comment (kiukchung) [inline kubernetes/README.md:35]: typo: `Ceate Amazon EKS` -> `AWS EKS`

File: kubernetes/README.md [hunk 2]
```diff
@@ -44,22 +46,36 @@ eksctl create cluster \
     --nodes=2
 ```
 
-Install Nvidia device plugin to enable GPU support on your cluster.
+##### Create Microsot AKS
```
Comment (kiukchung) [inline kubernetes/README.md:49]: typo: `Create Microsot AKS` -> `Microsoft AKS`

Comment (mszhanyi) [reply kubernetes/README.md]: Thank you very much. fixed.

File: kubernetes/README.md (hunk 1 above)
Comment (mszhanyi) [reply kubernetes/README.md]: Thank you! updated

File: kubernetes/README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ to setup a kubernetes cluster.
 >  you can use other Kubernetes clusters.
 
 #### Create your cluster
-##### Ceate Amazon EKS
+##### Ceate AWS EKS
```
Comment (kiukchung) [inline kubernetes/README.md:35]: typo: "Ceate" -> "Create"

**Pull request closed without merging**